### DISCUSSION TITLE
Sync Fork Message

### DISCUSSION THREAD

**Discussion by thecuriousteam** [CBS-India-Pvt-Ltd/Curious-Connect#52] [Announcements]
Please do sync fork

**Comment by thecuriousteam** [comment]
**Product - Curious Connect**

Team please take pull from *main branch*

Comment -  "footer setup done"

**GSSOC DEVELOPERS**
Please hit the *Sync Fork* in your personal repository and do *git pull*

**Reply by thecuriousteam** [reply]
Please do sync fork again

**Comment by thecuriousteam** [comment]
**Product - Curious Connect**

Team please take pull from *main branch*

Comment -  "meeting cards added"

*GSSOC DEVELOPERS*
Please hit the *Sync Fork* in your personal repository and do *git pull*

**Comment by thecuriousteam** [comment]
**Product - Curious Connect**

Team please take pull from *main branch*

Comment -  "footer enhancements"

*GSSOC DEVELOPERS*
Please hit the *Sync Fork* in your personal repository and do *git pull*

**Comment by thecuriousteam** [comment]
**Product - Curious Connect**

Team please take pull from *main branch*

Comment -  "features card added"

*GSSOC DEVELOPERS*
Please hit the *Sync Fork* in your personal repository and do *git pull*

**Comment by thecuriousteam** [comment]
**Product - Curious Connect**

Team please take pull from *main branch*

Comment -  "Join Room Initial Component Added"

*GSSOC DEVELOPERS*
Please hit the *Sync Fork* in your personal repository and do *git pull*

**Comment by deepakgupta-D3** [comment]
@thecuriousteam Hii, I want to contribute on this project , Please guide me

**Reply by thecuriousteam** [reply]
Yes @8deepakkumar8 you can start working on it
